kagenti/installer/app/checker.py

```diff
@@ -33,6 +33,10 @@ def check_dependencies():
     )
     all_ok = True
     for tool, versions in config.REQ_VERSIONS.items():
+        if tool == "docker" and tool != config.CONTAINER_ENGINE:
+            continue
+        if tool == "podman" and tool !=config.CONTAINER_ENGINE:
+            continue
```

Comment:
If CONTAINER_ENGINE is set to neither "docker" nor "podman", both checks will skip, and neither will be verified. Could be helpful to raise an error if an unsupported value is set. I think at this time we do need one or the other for the installation to work.

Reply:
Good point. I will raise an error if their options is selected.